FFmpegInterop/FFmpegInteropMSS.cpp

```diff
@@ -655,46 +690,48 @@ HRESULT FFmpegInteropMSS::InitFFmpegContext()
 		mss = ref new MediaStreamSource(videoStream->StreamDescriptor);
 		videoStream->EnableStream();
 	}
-	else
+	else if (subtitleStreams.size() == 0)
```

Comment:
Proposal: Change to
`else if (subtitleStreams.size() == 0 || !config->IsExternalSubtitleParser)`

Files with only subtitle streams should not be allowed, except for IsExternalSubtitleParser special case.

Reply:
Agreed.